hazelcast/src/main/java/com/hazelcast/map/impl/MapContainer.java

```diff
@@ -72,6 +73,9 @@
 @SuppressWarnings("WeakerAccess")
 public class MapContainer {
 
+    public static final HazelcastProperty LOAD_ALL_PUBLISHES_ADD_EVENT
```

Comment:
When should this property be enabled? How does an end user know this property needs to be set?

Reply:
Intention is to provide a soft pass. Because this is not a backward compatible change. This should be removed in 4.0. Doc update is needed to inform user.